### PR TITLE
Cloud Cover and standardized Container padding

### PR DESCRIPTION
**Pull request by tylersticka**
## Overview

This PR introduces a new `c-cloud-cover` component intended for introductory and page title content. It continues explorations that @aileenjeffries had done but uses CSS Grid Layout and viewport units to try some new hotness.

It also introduces new modifiers for `o-container` that establish fluid padding values to better coordinate between adjacent content areas. I also changed those stories a bit to include toggles for those modifiers.

Along the way I added https://github.com/postcss/postcss-media-minmax because the thought of maintaining separate tokens for `max-width` values made me feel weary.

(Note that there was a conflict in `package-lock.json` that accounts for the vast majority of the 70,000+ changes GitHub is saying this PR contains.)

### Narrow

Screen Shot 2020-11-12 at 10 45 11
Screen Shot 2020-11-12 at 10 45 19
Screen Shot 2020-11-12 at 10 45 26
Screen Shot 2020-11-12 at 10 45 35

### Wide

Screen Shot 2020-11-12 at 10 46 06
Screen Shot 2020-11-12 at 10 46 08
Screen Shot 2020-11-12 at 10 46 09
Screen Shot 2020-11-12 at 10 46 11

### CODE REVIEW

File: src/components/cloud-cover/demo/extra.twig [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+          level: -2,
+          content: 'What We Do'
+        } only %}
+        <p>
+          We
+          {% include '@cloudfour/components/icon/icon.twig' with {
+            name: 'heart',
+            inline: true
+          } only %}
+          <span class="u-hidden-visually">love</span>
```
Comment (Paul-Hebert) [inline src/components/cloud-cover/demo/extra.twig:17]: :+1:

**Review by Paul-Hebert — DISMISSED** [review]
Code looks good and the examples look great! Nice job!

**Comment by tylersticka** [comment]
@Paul-Hebert I had to resolve a `package-lock.json` conflict, could you re-review? No other changes made.

**Review by Paul-Hebert — APPROVED** [review]
Whoops, sorry I missed this ping!

**Comment by tylersticka** [comment]
@Paul-Hebert Thank you!

**Pull request merged**